README.md

```diff
@@ -108,7 +117,12 @@ java -jar target/quarkus-app/quarkus-run.jar \
     -p JBOSS_JIRA_PWD \
     start-release \
     -g <github org>/<github repo>/<git reference: branch, tag, hash> \
+    -ghu <Github user name>
     -o <github token> \
+    -glu <gitlab user> -glt <gitlab token> \
```

Comment:
I think the next improvement should be to move the user authentication setup to a file that can be consumed by the app without having to pass it around on the CLI.

Reply:
Created issue #135